src/main/scala/com/adevinta/unicron/artifactory/ArtifactorySettingsPlugin.scala

```diff
@@ -24,6 +24,8 @@ object ArtifactorySettingsPlugin extends AutoPlugin {
 
   val autoImport: ArtifactoryKeys.type = ArtifactoryKeys
 
+  override def trigger = allRequirements
```

Comment:
> By overriding trigger to allRequirements it is no longer needed to put a enablePlugins(ArtifactorySettingsPlugin) on each and every project (simply putting it in the plugins.sbt is enough).

can you update the README as well? because it still mentions you need to do that.